src/PsrHttpClient.php

```diff
@@ -35,4 +36,9 @@ public function request(PsrRequest $psrRequest, ?CancellationToken $cancellation
 
         return $this->psrAdapter->toPsrResponse($response);
     }
+
+    public function sendRequest(PsrRequest $request): PsrResponse
+    {
+        return $this->request($request);
+    }
```

Comment:
I guess we can just rename the above method instead of adding a second one?

Reply:
For some reason I thought the extra optional parameter would conflict - I'll fix that.

Reply:
It doesn't as long as it has a default value. 🙂